### DISCUSSION TITLE
BetterBird

### DISCUSSION THREAD

**Discussion by Patrxgt** [yokoffing/Betterfox#245] [Ideas]
It would be very nice to have something like this, but for Thunderbird. I'm using Betterfox on my Thunderbird instance and it's already great for my privacy, since Firefox and Thunderbird are very similar and have many similar about:config flags, but Thunderbird has some exclusive flags, that aren't covered by Betterfox.

**Comment by yokoffing** [comment]
They certainly have had a [resurgence](https://blog.thunderbird.net/2023/05/thunderbird-is-thriving-our-2022-financial-report/).

I’d like to see a lot of demand for it before I pour resources into it, but I’m open to the idea.

What prefs do you find useful there?

**Reply by Patrxgt** [reply]
>They certainly have had a [resurgence](https://blog.thunderbird.net/2023/05/thunderbird-is-thriving-our-2022-financial-report/).

Yeah, Thunderbird got quite a lot wind in it's wings after UI update.

>I’d like to see a lot of demand for it before I pour resources into it, but I’m open to the idea.

That's understandable. I hope, that other people would also like to see and use this kind of project!

>What prefs do you find useful there?

Pretty much everything, that isn't Firefox exclusive, so things that aren't referring to Firefox Pocket or OCSP or overall Firefox's interface.

**Reply by yokoffing** [reply]
Sure, Betterfox ESR could certainly be adapted for Thunderbird. But what Thunderbird-specific preferences, if any, have you found useful when using Thunderbird?

Are there any changes from the default Thunderbird settings that you would recommend?

**Reply by Patrxgt** [reply]
> Are there any changes from the default Thunderbird settings that you would recommend?

Not really. The only thing, that you could change for Betterfox is fixing blockage of some options, for example add-ons since they stopped working after using Betterfox and can't be even installed. But other than that, it's good.

Edit: I have fixed the add-on brakeage! You need to turn off requirement for XPI signatures or else Thunderbird will block all add-ons and themes you've installed and also installation of those.

**Reply by yokoffing** [reply]
Here is a project based on arkenfox and added Thunderbird-specific prefs: https://github.com/HorlogeSkynet/thunderbird-user.js/blob/824edabe6303d6b85a32fcba96901706ed4c5922/user.js#L1141

I haven't dedicated time to comb through it yet. I'll go through these recommendations soon -- only because I am trying out Thunderbird on a new Linux install.

Although, right now it's still based on Thunderbird 115 and not 128. But should still be of value.
